config/vendor-api-v1.yml

```diff
@@ -314,7 +314,7 @@ components:
           $ref: "#/components/schemas/ApplicationAttributes"
     ApplicationAttributes:
       type: object
-      additionalProperties: false
+      additionalProperties: true
```

Comment:
I think this isn't necessary because `hesa_itt_data` is allowed because it's in the `properties`.